src/ontology/mondo-edit.obo

```diff
@@ -330687,15 +330687,16 @@ property_value: confidence "0.0" xsd:double
 
 [Term]
 id: MONDO:0015012
-name: mucopolysaccharidosis-like syndrome with congenital heart defects and hematopoietic disorders
+name: mucopolysaccharidosis-plus syndrome
 subset: gard_rare {source="GARD:17944"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_malformation_syndrome
 subset: orphanet_rare {source="Orphanet:505248"}
 subset: rare
 synonym: "MPSPS" EXACT ABBREVIATION [OMIM:617303]
 synonym: "mucopolysaccharidosis-like plus disease" EXACT [Orphanet:505248]
-synonym: "mucopolysaccharidosis-plus syndrome" EXACT [OMIM:617303]
+synonym: "mucopolysaccharidosis-like syndrome with congenital heart defects and hematopoietic disorders" EXACT [Orphanet:505248]
+synonym: "mucopolysaccharidosis-plus syndrome" EXACT CLINGEN_PREFERRED [OMIM:617303]
```

Comment:
@nicolevasilevsky Should we add all the PMID mentioned in the issue?

Reply:
yes! I missed that